server/api/models/recipe.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const schema = new Schema(
+  {
+    servings: {
+      type: Number,
+      required: true
+    },
+    prepTime: String,
+    cookTime: String,
+    ingredients: [{
+      measurement: {
+        type: String,
+        enum: ["grams", "tablespoons", "count"], // TODO: Comprehensive enum list
+        required: true
+      },
+      quantity: {
+        type: Number,
+        required: true
+      },
+      name: {
+        type: String,
+        required: true
+      },
+    }],
+    steps: [{
+      type: String,
+      required: true
+    }],
+    links: [String],
+    name: {
+      type: String,
+      required: true
+    },
+    // image: {
+    //   type: String,
+    //   required: true // TODO: Figure out how to store images
+    // },
```

Comment:
TODO: Either remove this or put in a real schema field.

We can either store URLs to images or we can store the images in the DB. Might be worth spiking this.

Reply:
Yeah lets just remove this for now, we should spike this.